ruby-challenges.rb

```diff
@@ -31,3 +55,15 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome string
+  if string.upcase.reverse == string.upcase
```

Comment:
definitely a key step is to make sure that they are all upper or lower case. Great work!